volttron/utils/rmq_setup.py

```diff
@@ -659,14 +659,12 @@ def _create_rabbitmq_config(rmq_config, setup_type):
                     while True:
                         prompt = 'Enter the root CA certificate public key file:'
                         root_public = prompt_response(prompt, mandatory=True)
-                        root_public = os.path.expanduser(root_public)
                         if is_file_readable(root_public):
```

Comment:
Why are you not continuing to expanduser (should be expandall)?  root_public unless I am missing something will never get expanded to the full path.  The is_file_readable function does the expansion, but the passed string is not changed and it shouldn't be.

Reply:
https://volttron.readthedocs.io/en/develop/_modules/volttron/platform/config.html#expandall